### PR TITLE
Compile wasm to wast on the frontend

### PR DESCRIPTION
**Pull request by hugo-dc**
Compile wasm to wast on the frontend as required in https://github.com/ewasm/etherchain-light/issues/27

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,6 @@
     "solc": "^0.4.9",
     "string-similarity": "^1.1.0",
     "tmp": "0.0.31",
-    "wabt": "^1.0.0",
```
Comment (axic) [inline package.json:29]: You could still use this dependency system and add a `prepublish` (or in recent npm: `prepare`) step which copies the file.

Comment (cdetrio) [reply package.json:29]: also note that wabt 1.0.5 broke the `readWasm` function.  so pin the dependency to wabt 1.0.0.

Comment (cdetrio) [reply package.json:29]: I looked for 90 seconds but didn't find decent documentation on how to use `npm prepare` so not bothering with that now.

**Review by cdetrio — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (axic) [reply package.json:29]: `prepare` is the new name of `prepublish`, nothing else changed (well, they split `prepublish` into two parts: run during build vs. run before npm publish)

**Comment by hugo-dc** [comment]
@axic I'm working on using `prepare`

**Pull request merged**